CHANGELOG.rst

```diff
@@ -4,6 +4,11 @@ Community Healthchecks.io Release Notes
 
 .. contents:: Topics
 
+### [Unreleased]
```

Comment:
As mentioned in the comments, please revert this and create a changelog fragment instead.

Reply:
Reverted the changelog and created the fragment.

https://github.com/ansible-collections/community.healthchecksio/pull/32/files#diff-74367a133dad97faec75d28dc8af30e4410563e179ca8f2acfc115698ab8f63b